src/routes/lobby/LobbyView.vue

```diff
@@ -128,6 +130,9 @@ export default {
     gameName() {
       return this.gameStore.name;
     },
+    opponentUsername() {
+      return this.gameStore?.opponentUsername;
+    },
```

Comment:
It looks like you defined this computed prop without actually consuming it in the template code. Also do we need to optionally chain here? The gameStore should always be defined

Reply:
@JonasThers you're right it makes sense why we'd need the computed prop with the way the watcher is written, now I see the problem. Can you please additionally refactor the instance of the opponent's player indicator to use the computed prop on line 26?

Reply:
This value feels like it should be mapped in instead of defined this way, though I agree optionally chaining is not required here.

Reply:
@JonasThers not a stupid question. Pinia has `mapState()` and `mapGetters()` helpers that components can use to define computed properties like this that just return a given value from the store. We have several examples in the codebase you can find by searching for mapState and mapGetters